pymc3/distributions/dist_math.py

```diff
@@ -365,6 +365,7 @@ def conjugate_solve_triangular(outer, inner):
             grad = tt.triu(s + s.T) - tt.diag(tt.diagonal(s))
         return [tt.switch(ok, grad, floatX(np.nan))]
 
+
 class SplineWrapper (theano.Op):
```

Comment:
I've fixed all these issues. But just to argue about PEP8... :)

PEP8 forbids spaces after functions or methods, but doesn't say about a whitespace before the base class:
> Avoid extraneous whitespace in the following situations:
> ... (nothing interesting)
> Immediately before the open parenthesis that starts the argument list of a function call
> ... (nothing interesting)

Also it says that
> Method definitions inside a class are surrounded by a single blank line

so a blank line before the first method in a class is legitimate as it surrounds a class method, it is even possible to argue that it is required by PEP8 there.

Actually `pep8` linter seems to agree with both ways:

```sh
$ cat t.py 
class Foo (object):

    def foo(self):
        print('hi')


class Bar(object):
    def bar(self):
        print('hi')
$ pep8 t.py
$ # no errors reported
```

And as PEP8 is a formalized description of the code style used in Python standard library, we can check what is used in it, and it turns out that both variants are present there:

*  [`class ZipInfo (object)`](https://github.com/python/cpython/blob/master/Lib/zipfile.py#L294)
and [`class LZMAFile(_compression.BaseStream)`](https://github.com/python/cpython/blob/master/Lib/lzma.py#L38), although `class Foo(object)` is more frequent than `class Foo (object)`;

* [`class _Outcome(object):\n    def __init__(self, result=None):`](https://github.com/python/cpython/blob/master/Lib/unittest/case.py#L44) and [`class _BaseTestCaseContext:\n\n    def __init__(self, test_case):`,](https://github.com/python/cpython/blob/master/Lib/unittest/case.py#L128) although

    ```python
    class Foo:

        def __init__(...):
    ```

    is more frequent than
    ```python
    class Foo:
       def __init__(...):
       ...
    ```

However I've already commited the fixes and I see that we need to not just follow PEP8, but also be consistent with other parts of PyMC3 :)

As an offtopic, I actually think that it would be nice to have `pep8` or `flake8` running automatically in Travis and failing builds if case of PEP8 violations. However I briefly investigated this possibility and it seems like it is necessary to either fix some rather large parts of the existing code and/or agree on exceptions of PEP8 that we don't want to follow, like for example maximum line length.